.github/workflows/wan-perf.yml

```diff
@@ -1,23 +1,26 @@
 name: WAN Perf
 
+#on:
+#  push:
+#    branches:
+#    - main
+#    paths:
+#    - .github/workflows/wan-perf.yml
+#    - src/core/*
+#    - src/platform/*
+#    - src/perf/*
+#  pull_request:
+#    branches:
+#    - main
+#    paths:
+#    - .github/workflows/wan-perf.yml
+#    - src/core/*
+#    - src/platform/*
+#    - src/perf/*
+#    - submodules/openssl/*
```

Comment:
Instead of making this entirely separate, just keep it with the `on:` below, after the workflow_dispatch, but still commented out.